### PR TITLE
Add a task to train invalid_compatibility_report model

### PR DESCRIPTION
**Pull request by ksy36**
Fixes https://github.com/mozilla/bugbug/issues/3838

### CODE REVIEW

File: bugbug/models/invalid_compatibility_report.py [hunk 1]
```diff
@@ -50,7 +50,8 @@ def __init__(self, lemmatization=False):
             ]
         )
 
-        self.clf = xgboost.XGBClassifier(n_jobs=utils.get_physical_cpu_count())
+        self.hyperparameter = {"n_jobs": utils.get_physical_cpu_count()}
+        self.clf = xgboost.XGBClassifier(**self.hyperparameter)
```
Comment (ksy36) [inline bugbug/models/invalid_compatibility_report.py:55]: (noticed this was added to all models and the training task fails without it)

Comment (suhaibmujahid) [reply bugbug/models/invalid_compatibility_report.py:55]: Could you please submit it in a separate PR?

Comment (ksy36) [reply bugbug/models/invalid_compatibility_report.py]: Yeah, sounds good!

**Review by marco-c — APPROVED** [review]
LGTM!

**Comment by marco-c** [comment]
I will make a release and trigger the hook so we can see the task in action.

**Comment by ksy36** [comment]
Thanks!

**Comment by marco-c** [comment]
Here's the training task: https://community-tc.services.mozilla.com/tasks/GtGcPynVQB63abxYQksHjA/runs/0.

**Pull request merged**